devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus-jbang/code/jbang-picocli-code/codestart.yml

```
@@ -0,0 +1,19 @@
+name: jbang-picocli-code
+ref: picocli
+type: code
+language:
+  base:
+    data:
+      resource:
```

That maybe better since it's not a resource:
```suggestion
      command:
```

---

You will need to update when using it of course..